### ISSUE TITLE
Can't stop realtime parsing (again)

### ISSUE THREAD

**Issue by Zarainia** [RedFantom/GSF-Parser#88]
After being started, realtime parsing cannot be stopped. The following error is shown when the button is clicked again. 

![error](https://user-images.githubusercontent.com/14959769/26886473-791b5708-4b73-11e7-8634-7af4b4d1a103.JPG)

It seems that in `start_parsing` in `RealtimeFrame`, `variables.settings_obj.screenparsing` is not true the first time when the parsing is started, so the queues are never made.

**Comment by RedFantom** [comment]
Yes, that's correct. It appears I have already fixed this in the `screen` branch, with a simple `if` statement:
```python
if self.screenparser:
    self.exit_queue.put(False)
```
I did not realize this issue was also apparent in the `master` branch, but now I think about it, that's actually very logical. Thank you for pointing it out, I will double my efforts to make the `screen` branch mergeable ASAP by finishing that darn build calculator.
